src/Chainweb/Chainweb.hs

```diff
@@ -271,6 +273,7 @@ defaultChainwebConfiguration v = ChainwebConfiguration
     , _configPruneChainDatabase = True
     , _configBlockGasLimit = 100000
     , _configCutFetchTimeout = 10000000
+    , _configCutLimitConfig = LimitConfig 60 60 30 240
```

Comment:
What does this mean? Documentation?

Reply:
```haskell
data LimitConfig = LimitConfig
    { maxBucketTokens :: {-# UNPACK #-} !Int
    , initialBucketTokens :: {-# UNPACK #-} !Int
    , bucketRefillTokensPerSecond :: {-# UNPACK #-} !Int
    , badSenderPenaltySecs :: {-# UNPACK #-} !Int
    } deriving (Eq, Ord, Show)
```

Reply:
We should use the the fields when constructing a `LimitConfig` value.

Also are these values exposed in the configuration file? If that's the case:

1. isn't this to complex for a normal user to handle?

2. should we hide this from the user? (not sure if hidden config options are good, because we would have to document them somewhere carefully, to not forget about them)

3. Does it have a proper syntax?

Reply:
> Also are these values exposed in the configuration file?

Yes, at least in this PR.

> isn't this to complex for a normal user to handle?

Likely, yes. We wanted it fully configurable at the time for rapid testing.

> should we hide this from the user?

Probably. @gregorycollins ?

> Does it have a proper syntax?

I gave it a JSON instance.